simEarth_fork.py

```diff
@@ -0,0 +1,55 @@
+import numpy as np
+
+#state = np.array([100, 200, 130, 1.05, 1.02, 0.75])
+
+    state = {"pop": 100 , "fos"; 200, "fol": 130, "bir": 1.05, "der": 1.02, "fef": 0.75}
+
+def function_food_growth(array):
```

Comment:
`array` is probably a bad name. In this case this isn't even an array. What's the semantic meaning for this "array"?